components/observability/org.wso2.vick.tracing.receiver/src/main/thrift/zipkincore.thrift

```diff
@@ -0,0 +1,344 @@
+# Copyright 2012 Twitter Inc.
```

Comment:
Shall we double check on the license header.

Reply:
This thrift file is taken from [Zipkin Specification](https://github.com/openzipkin/zipkin-api/blob/master/thrift/zipkinCore.thrift) which already has a license header. Hence the twitter license. These were added because the thrift Java classes were to be automatically generated at build time.

However, as per the latest changes we do not generate classes at build time. Therefore we can remove this file completely. I will add a PR to remove.

Reply:
Fixed with #172